spec/brainstem/dsl/field_spec.rb

```diff
@@ -99,4 +99,31 @@ def matches?(model, helper_instance, conditional_cache)
       end
     end
   end
+
+  describe "#optioned?" do
+    context "when is not an optional field" do
+      it 'returns true' do
+        expect(field.optioned?(['expensive_title', 'expensive_title2'])).to eq true
+      end
```

Comment:
Actually, now that I look at the actual spec helper, it has `optional: true` for both of those.  Is this test returning true because those are optional fields?